harmonica/isostasy.py

```diff
@@ -0,0 +1,69 @@
+"""
+Function to calculate the thickness of the roots and antiroots assuming the
+Airy isostatic hypothesis.
+"""
+import numpy as np
+
+
+def isostasy_airy(
+    topography,
+    density_upper_crust,
+    density_lower_crust,
+    density_mantle,
+    density_water=None,
+):
+    """
+    Computes the thickness of the roots/antiroots using the Airy's hypothesis.
```

Comment:
```suggestion
    Computes the thickness of the roots/antiroots using Airy's hypothesis.
```

Nice! I like this wording a lot better. Avoids too many assumptions and is the way we see it written on text books.